frontend/src/stylesheets/Authentication.scss

```diff
@@ -99,7 +98,6 @@ $auth-dark: #d9535d;
 }
 
 .Auth-link {
-  text-align: center;
```

Comment:
To chyba też nie powinno być usunięte, do potwierdzenia z @agngan.

Reply:
to po prostu przywrócę, usunąłem w sumie niechcaćy